actor/builtin/storagemarket/storagemarket.go

```diff
@@ -237,6 +288,19 @@ func (sma *Actor) GetProofsMode(vmctx exec.VMContext) (types.ProofsMode, uint8,
 	return size, 0, nil
 }
 
+func (sma *Actor) getMinerIsProving(vmctx exec.VMContext, minerAddr address.Address) (bool, error) {
+	msgResult, _, err := vmctx.Send(minerAddr, "isProving", types.ZeroAttoFIL, nil)
+	if err != nil {
+		return false, err
+	}
+
+	isProving, err := abi.Deserialize(msgResult[0], abi.Boolean)
+	if err != nil {
+		return false, err
```

Comment:
Blocking: This needs to be wrapped in a fault errror

Reply:
GetProofsMode and GetTotalStorage don't wrap in fault errors -- should these be updated as well?